ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/disconnected/provision/cucushift-installer-rehearse-vsphere-upi-disconnected-provision-chain.yaml

```diff
@@ -5,7 +5,9 @@ chain:
   - ref: openshift-cluster-bot-rbac
   - ref: ipi-conf
   - ref: ipi-conf-vsphere-check
-  - ref: ipi-conf-vsphere-mirror-image
+  - ref: ipi-conf-vsphere-mirror-registry
```

Comment:
If possible, update the steps to replace `ref: ipi-conf-vsphere-mirror-registry` with `chain: ipi-conf-vsphere-mirror-registry`

Reply:
Both `ref: ipi-conf-vsphere-mirror-registry` and `ref: mirror-images-by-oc-adm` need `ref:ipi-install-rbac`, so add `ref:ipi-install-rbac` before them.
Do you mean that add new `chain: ipi-conf-vsphere-mirror-registry` including  `ref: ipi-conf-vsphere-mirror-registry` and `ref:ipi-install-rbac`?

Reply:
Ah, sorry, ignore my last comments, I meant:
If possible, update the steps to replace `ref: mirror-images-by-oc-adm` with `chain: mirror-images-by-oc-adm`.

Based on your last comment, if `ipi-conf-vsphere-mirror-registry` also need `ref: ipi-install-rbac`, then it is okay to keep the current step list.